### PR TITLE
kconf: init at 1.10.1

### PR DESCRIPTION
**Pull request by thmzlt**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -30461,4 +30461,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  kconf = callPackage ../applications/networking/cluster/kconf { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:30465]: Please sort this entry into a category and there alphabetically.

Comment (thmzlt) [reply pkgs/top-level/all-packages.nix:30465]: I tried to find a pattern, but there are networking/cluster packages spread across the entire file. Can you point to where this entry should go?

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:30465]: There should be a headline like ```### NETWORK`` and yeah we need to programmatically sort this.

Comment (thmzlt) [reply pkgs/top-level/all-packages.nix]: I couldn't find a `### NETWORK` header, but I moved it near some other Kubernetes-related tools. Is this better?

**Comment by thmzlt** [comment]
@SuperSandro2000, would you mind taking another look at this? Thanks.

**Pull request merged**